checkov/secrets/plugins/load_detectors.py

```diff
@@ -86,3 +86,23 @@ def transforms_policies_to_detectors_list(custom_secrets: List[Dict[str, Any]])
         if not parsed:
             logging.info(f"policy : {secret_policy} could not be parsed")
     return custom_detectors
+
+
+def get_runnable_plugins(policies: List[Dict[str, Any]]) -> Dict[str, bytes]:
+    runnables: dict[str, bytes] = {}
+    for policy in policies:
+        code = policy['code']
+        if code:
+            try:
+                code_dict = yaml.safe_load(code)
+                if 'definition' in code_dict:
+                    if 'is_runnable' in code_dict['definition'] and 'value' in code_dict['definition']:
+                        encoded_payload = code_dict['definition']['value']
+                        if isinstance(encoded_payload, list):
+                            encoded_payload = encoded_payload[0]
+                        decoded_payload = decompress_file_gzip_base64(encoded_payload)
+                        name: str = policy['title']
+                        runnables[name] = decoded_payload
+            except Exception as e:
+                logging.warning(f"Could not parse runnable policy {policy['title']} due to: {e}")
```

Comment:
This shouldn't spam logs. The yaml load is safe since we already do it in the secrets flow in another place and conf_incidents code field must be a valid yaml. The only other thing that can crash is the decoding process but that happens only for runnables (of which we have only one).

Reply:
Why would all the rest of the non OS policies not go through this flow, and throw warning logs?

Reply:
Because they aren't marked as 'runnable'.
The only thing they go through is yaml loading but that is already happening now without problems in `add_detectors_from_code` so it is a safe operation.